src/common/cockpitwebcertificate.c

```diff
@@ -117,7 +118,7 @@ cockpit_certificate_locate (bool missing_ok,
         return cert_path;
     }
 
-  if (error && !missing_ok)
```

Comment:
okay.  This one is really wrong, though.  We just added an assert for `error != NULL` above.  Surely when we re-run the scan, it'll be happy enough now?

Reply:
I'm afraid I don't understand -- coverity's objection was that `error &&` is always true here, so it's a dead condition.

Reply:
Uh.  `EBRAIN`.